compoments/report-repair/repair-availability.js

```diff
@@ -14,7 +14,7 @@ const RepairAvailability = ({handleChange, values, nextAvailability}) => {
     repairLocation:  values.repairLocation,
     repairProblem:  values.repairProblem,
     repairIssue: values.repairProblemBestDescription,
-    uprn: values.uprn,
+    locationId: values.address,
```

Comment:
I think `values.address` won't be the `locationId` required by the API - it's value is currently the lines of the address.

![image](https://user-images.githubusercontent.com/1875867/145446879-abc3f1e0-b262-48e2-a0f3-7fb1c88f9689.png)

Have I missed the change that sets `values.address` to an ID?

I believe that the `Address` component needs to be updated to store the id and the address value in the state.

Reply:
After updating the API to return `locationId` the value of address is now prefixed with the locationId:
![image](https://user-images.githubusercontent.com/1875867/145453999-9f68cc28-70df-4ee6-9b06-a3e651619b46.png)

...it should be separated to 2 distinct properties of the `address` value in the state.